business-registry-dashboard/app/app.config.ts

```diff
@@ -247,15 +248,15 @@ export default defineAppConfig({
     tooltip: {
       wrapper: 'relative inline-flex max-h-min',
       container: 'z-20 group min-h-fit',
-      background: 'bg-gray-700',
+      background: 'bg-gray-700/90',
```

Comment:
Adding the 90% opacity. From the UXA notes ticket:
![image](https://github.com/user-attachments/assets/4e622931-dfc6-432b-ac4c-1f767854c6a5)